### PR TITLE
Reduce timeout for lb and storage test

### PR DESCRIPTION
**Pull request by alvaroaleman**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:

**Release note**:

```release-note
none
```

/assign @mrIncompetent 

/CC @nikhita @p0lyn0mial

### CODE REVIEW

File: api/cmd/conformance-tests/main.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ type secrets struct {
 }
 
 const (
-	defaultTimeout                 = 30 * time.Minute
+	defaultTimeout                 = 10 * time.Minute
```
Comment (p0lyn0mial) [inline api/cmd/conformance-tests/main.go:109]: I like when we decrease timeout :)

**Comment by alvaroaleman** [comment]
@mrIncompetent Can you approve? The failed test is non-blocking I think as it doesn't exist anymore because it was moved to a periodic

**Pull request merged**